pandas/tests/extension/test_categorical.py

```diff
@@ -25,7 +25,12 @@
 
 
 def make_data():
-    return np.random.choice(list(string.ascii_letters), size=100)
+    while True:
+        values = np.random.choice(list(string.ascii_letters), size=100)
+        # ensure we meet the requirement
```

Comment:
no repeated values but duplicates allowed?

Reply:
Just that the first two are distinct., since the `where` test requires that `data[0] != data[1]`.